examples/pinocchio/Readme.md

```diff
@@ -0,0 +1,37 @@
+# [WIP]
+# Lambdaworks Pinocchio 
+
+This is an implementation of [Pinocchio protocol](https://eprint.iacr.org/2013/279.pdf) using Lambdaworks. This sourse code is the companion of this [blog post](link) aimed at those who want to learn about SNARKs.
+
+# Understanding the code
+We encourage to start by reading the [blog post](link) to understand the code.
```

Comment:
I suggest updating the link with the one the post will have when published.

Reply:
source code